src/Microsoft.AspNetCore.Razor.Sdk/SDK/Sdk.targets

```diff
@@ -9,12 +9,13 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
 
-  <Import Project="$(_RazorSdkCurrentVersionPath)"
-    Condition="'$(_RazorSdkCurrentVersionPath)' != '' AND Exists('$(_RazorSdkCurrentVersionPath)')" />
+  <PropertyGroup Condition="'$(RazorSdkTargetsPath)' == ''">
+    <RazorSdkTargetsPath Condition="'$(IsCrossTargetingBuild)' == 'true'">$(MSBuildThisFileDirectory)..\buildMultiTargeting\Razor.Sdk.CurrentVersion.MultiTargeting.targets</RazorSdkTargetsPath>
+    <RazorSdkTargetsPath Condition="'$(IsCrossTargetingBuild)' != 'true'">$(MSBuildThisFileDirectory)..\build\netstandard2.0\Razor.Sdk.CurrentVersion.targets</RazorSdkTargetsPath>
+  </PropertyGroup>
 
-  <Import Project="$(MSBuildThisFileDirectory)..\build\netstandard2.0\Razor.Sdk.CurrentVersion.targets"
```

Comment:
I removed `MicrosoftAspNetCoreRazorSdkTargetsImported` and made `RazorSdkTargetsPath`. Sort of what Microsoft.CSharp.Targets does